eagle-core/eagle-alert-parent/eagle-alert/alert-engine/src/main/java/org/apache/eagle/alert/engine/runner/AlertBolt.java

```diff
@@ -111,9 +111,12 @@ public void execute(Tuple input) {
                     pe.getEvent().setMetaVersion(specVersion);
                 }
                 // check if specVersion is older than stream_event_version
-                else if (specVersion != null && stream_event_version != null && specVersion.contains("spec_version_") && specVersion.contains("spec_version_")){
-                    Long timestamp_of_specVersion = Long.valueOf(specVersion.split("spec_version_")[1]);
-                    Long timestamp_of_streamEventVersion = Long.valueOf(stream_event_version.split("spec_version_")[1]);
+                else if (specVersion != null && stream_event_version != null &&
+                        specVersion.contains("spec_version_") && stream_event_version.contains("spec_version_")){
+//                    Long timestamp_of_specVersion = Long.valueOf(specVersion.split("spec_version_")[1]);
+//                    Long timestamp_of_streamEventVersion = Long.valueOf(stream_event_version.split("spec_version_")[1]);
+                    Long timestamp_of_specVersion = Long.valueOf(specVersion.substring(13));
```

Comment:
What I got for "specVersion" is "spec_version_1471478137249". split[1] would be out of array index.
So I just substring the timestamp.